flask_lesson_intro/utils.py

```diff
@@ -0,0 +1,6 @@
+import json
+
+
+def get_data():
+    with open("data.json") as file:
```

Comment:
if a file is not present or the file contains not valid data we should catch an error